messages/npm.dependencies.pin.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "lock a list of dependencies to a target release or default to 'latest', place these entries in 'packagesToHardcodeVersions' entry in the package.json",
```

Comment:
This needs to be updated to `target tag` and the right property in the package.json